src/vivarium_census_prl_synth_pop/data/loader.py

```diff
@@ -112,12 +112,19 @@ def load_persons(key: str, location: str) -> pd.DataFrame:
     data.relation_to_household_head = data.relation_to_household_head.map(
         metadata.RELATIONSHIP_TO_HOUSEHOLD_HEAD_MAP
     )
-
-    # put all non-draw columns in the index, else vivarium will drop them
-    data = data.set_index(
-        ["census_household_id", "age", "relation_to_household_head", "sex", "race_ethnicity"]
+    # Map native born persons and if person has migrated in last year
+    data["born_in_us"] = data["born_in_us"].map(
+        metadata.NATIVITY_MAP
+    )  # True for native citizens
```

Comment:
This comment is a bit inaccurate/confusing because nativity != citizenship.

Reply:
Good catch.